### ISSUE TITLE
Can't find "cholmod.h" while installing using pip on macOS

### ISSUE THREAD

**Issue by QYQSDTC** [scikit-sparse/scikit-sparse#67]
I am installing scikit-sparse via pip and I got the error below:
```
 Building wheel for scikit-sparse (PEP 517) ... error
  ERROR: Command errored out with exit status 1:
   command: /opt/local/Library/Frameworks/Python.framework/Versions/3.8/bin/python3.8 /opt/local/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py build_wheel /var/folders/zl/498rsyv11_v53_k5x1xfflkw0000gn/T/tmphqddsg3a
       cwd: /private/var/folders/zl/498rsyv11_v53_k5x1xfflkw0000gn/T/pip-install-0ezcix_f/scikit-sparse_63d2a4408dfe4ad1a24ef3837a8eea73
  Complete output (35 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-11.3-x86_64-3.8
  creating build/lib.macosx-11.3-x86_64-3.8/sksparse
  copying sksparse/__init__.py -> build/lib.macosx-11.3-x86_64-3.8/sksparse
  copying sksparse/test_cholmod.py -> build/lib.macosx-11.3-x86_64-3.8/sksparse
  creating build/lib.macosx-11.3-x86_64-3.8/sksparse/test_data
  copying sksparse/test_data/illc1033.mtx.gz -> build/lib.macosx-11.3-x86_64-3.8/sksparse/test_data
  copying sksparse/test_data/illc1033_rhs1.mtx.gz -> build/lib.macosx-11.3-x86_64-3.8/sksparse/test_data
  copying sksparse/test_data/well1033_rhs1.mtx.gz -> build/lib.macosx-11.3-x86_64-3.8/sksparse/test_data
  copying sksparse/test_data/well1033.mtx.gz -> build/lib.macosx-11.3-x86_64-3.8/sksparse/test_data
  copying sksparse/test_data/well1850_rhs1.mtx.gz -> build/lib.macosx-11.3-x86_64-3.8/sksparse/test_data
  copying sksparse/test_data/illc1850.mtx.gz -> build/lib.macosx-11.3-x86_64-3.8/sksparse/test_data
  copying sksparse/test_data/illc1850_rhs1.mtx.gz -> build/lib.macosx-11.3-x86_64-3.8/sksparse/test_data
  copying sksparse/test_data/well1850.mtx.gz -> build/lib.macosx-11.3-x86_64-3.8/sksparse/test_data
  running build_ext
  building 'sksparse.cholmod' extension
  creating build/temp.macosx-11.3-x86_64-3.8
  creating build/temp.macosx-11.3-x86_64-3.8/sksparse
  /usr/bin/clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -pipe -Os -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX11.3.sdk -Isksparse -I/private/var/folders/zl/498rsyv11_v53_k5x1xfflkw0000gn/T/pip-build-env-o0pnraoh/overlay/lib/python3.8/site-packages/numpy/core/include -I/opt/local/Library/Frameworks/Python.framework/Versions/3.8/include -I/usr/include/suitesparse -I/opt/local/Library/Frameworks/Python.framework/Versions/3.8/include/python3.8 -c sksparse/cholmod.c -o build/temp.macosx-11.3-x86_64-3.8/sksparse/cholmod.o
  In file included from sksparse/cholmod.c:638:
  In file included from /private/var/folders/zl/498rsyv11_v53_k5x1xfflkw0000gn/T/pip-build-env-o0pnraoh/overlay/lib/python3.8/site-packages/numpy/core/include/numpy/arrayobject.h:4:
  In file included from /private/var/folders/zl/498rsyv11_v53_k5x1xfflkw0000gn/T/pip-build-env-o0pnraoh/overlay/lib/python3.8/site-packages/numpy/core/include/numpy/ndarrayobject.h:12:
  In file included from /private/var/folders/zl/498rsyv11_v53_k5x1xfflkw0000gn/T/pip-build-env-o0pnraoh/overlay/lib/python3.8/site-packages/numpy/core/include/numpy/ndarraytypes.h:1830:
  /private/var/folders/zl/498rsyv11_v53_k5x1xfflkw0000gn/T/pip-build-env-o0pnraoh/overlay/lib/python3.8/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: "Using deprecated NumPy API, disable it with "          "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-W#warnings]
  #warning "Using deprecated NumPy API, disable it with " \
   ^
  In file included from sksparse/cholmod.c:640:
  sksparse/cholmod_backward_compatible.h:1:10: fatal error: 'cholmod.h' file not found
  #include "cholmod.h"
           ^~~~~~~~~~~
  1 warning and 1 error generated.
  error: command '/usr/bin/clang' failed with exit status 1
  ----------------------------------------
 ERROR: Failed building wheel for scikit-sparse
Failed to build scikit-sparse
ERROR: Could not build wheels for scikit-sparse which use PEP 517 and cannot be installed directly
```
And I already installed suitesparse via MacPorts, it seems it is trying to find "cholmod.h" in `/usr/include` which does not even exist, and the suitesparse files are in `/opt/local/include`. Is there a way to modify the include path for pip?
![image](https://user-images.githubusercontent.com/15608306/120813832-dd0c7900-c580-11eb-9dd9-3b82fa21cf3b.png)

**Comment by QYQSDTC** [comment]
Fixed by referring from [the last answer](https://stackoverflow.com/questions/19456999/failing-to-install-scikits-sparse-clang-error), then build manually.

**Comment by jellis18** [comment]
This should be solved by #71
